include/alpaka/core/OmpSchedule.hpp

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2020 Sergei Bastrakov
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/core/Common.hpp>
+
+#ifdef _OPENMP
+#    include <omp.h>
+#endif
+
+#include <cstdint>
+
+
+namespace alpaka
+{
+    namespace omp
+    {
+        //#############################################################################
+        //! Representation of OpenMP schedule information: kind and chunk size.
+        //!
+        //! This class can be used regardless of whether OpenMP is enabled
+        struct Schedule
+        {
+            //#############################################################################
+            //! Mirror of the OpenMP enum omp_sched_t with the corresponding integer values
+            enum Kind
+            {
+                Static = 1,
+                Dynamic = 2,
+                Guided = 3,
+                Auto = 4
+            };
+
+            //! Integer representation of schedule kind.
+            //!
```

Comment:
I'm now confused. Isn't clang-format taking care of that?

Reply:
Could you clarify a bit? What should clang-format do here?

Reply:
I was not sure how to format this comment, i.e. whether to separate comment "head" with an empty line. And thought it would be auto-formatted. But I guess that was just stupid thinking on my side